tests/test_endpoints.py

```diff
@@ -22,3 +24,28 @@ class TestGetAllTables(TestCase):
     def test_all_returned(self):
         tables = get_all_tables([TableC])
         self.assertEqual(tables, [TableC, TableB, TableA])
+
+
+class TestAdminRouter(TestCase):
+    def test_get_meta(self):
+        client = TestClient(APP)
+
+        response = client.get("/meta/")
+        self.assertEqual(
+            response.json(),
+            {"piccolo_admin_version": "0.14.0", "site_name": "Piccolo Admin"},
```

Comment:
This will break in future versions of Piccolo admin.

We might have to do

```python
from piccolo_admin.version import __VERSION__

{"piccolo_admin_version": __VERSION__, "site_name": "Piccolo Admin"},
```

Reply:
I'll push this change.